<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="currentIndex">
   <number>5</number>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <property name="elideMode">
   <enum>Qt::ElideRight</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabsClosable">
   <bool>false</bool>
  </property>
  <property name="movable">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="Connection">
   <attribute name="title">
    <string>Connection</string>
   </attribute>
   <widget class="QPushButton" name="BConnect">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>36</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BDisconect">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnecte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BConectRafr">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rafraichir</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CBport">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="oprint">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="title">
    <string>Print</string>
   </attribute>
   <widget class="QPushButton" name="BPrint">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BLoadPrint">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>911</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>75</number>
    </property>
   </widget>
   <widget class="QListWidget" name="LWGcode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BPlay">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BPause">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BStop">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_175.png</normaloff>glyphicons_175.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BRafrGcode">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_082.png</normaloff>glyphicons_082.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>99</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_022_fire.png</normaloff>glyphicons_022_fire.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="otemp">
   <attribute name="title">
    <string>Temp√©rature</string>
   </attribute>
   <widget class="QLineEdit" name="LETemExt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LETembed">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BSetExtru">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extruder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BSetBed">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Bed</string>
    </property>
   </widget>
   <widget class="QLabel" name="LTemEx">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Actuel :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LTemExC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cible :  0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LTemBed">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Actuel :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LTemBedC">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cible :  0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="ocontrole">
   <attribute name="title">
    <string>Controle</string>
   </attribute>
   <widget class="QPushButton" name="BZZU">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BZU">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BU">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BD">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BC">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BX">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BY">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BZ_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BHX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_020_home.png</normaloff>glyphicons_020_home.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BHY">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_020_home.png</normaloff>glyphicons_020_home.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BHZ">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>glyphicons_020_home.png</normaloff>glyphicons_020_home.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BExtru">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extrude</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BEC">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BED">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BECent">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BRetra">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Retract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BMotor">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Moteur Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFOFF">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fan off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFan">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Ventilation</string>
    </property>
   </widget>
   <widget class="QSlider" name="SFan">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SVitesse">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SliderFlux">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="sliderPosition">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>390</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BVitesse">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFlux">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flux</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BX_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X-</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BY_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BZ">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="LTFan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LTVitesse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="LTFlux">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="ocam">
   <attribute name="title">
    <string>Cam√©ra</string>
   </attribute>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="url">
     <url>
      <string>http://127.0.0.1:8080/?action=snapshot</string>
     </url>
    </property>
   </widget>
   <widget class="QLabel" name="LTime">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Timelaps On/Off</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BSaveTime">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauver</string>
    </property>
   </widget>
   <widget class="QLabel" name="LDerImg">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>290</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Derniere image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LFps">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>fps</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBParDef">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Par defaut</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEFps">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEDerImg">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="CBTime">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BRafCam">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Rafraichir</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="oterminal">
   <attribute name="title">
    <string>Terminal</string>
   </attribute>
   <widget class="QLineEdit" name="LETerminal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTerminal">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QListWidget" name="LWTerminal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>981</width>
      <height>431</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
